src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -12,7 +12,8 @@ export interface ParamsForProvider {
 
 export default class PaypalCommerceRequestSender {
     constructor(
-        private _requestSender: RequestSender
+        private _requestSender: RequestSender,
+        private _isAPM?: boolean | undefined
```

Comment:
you don't need ` | undefined` if it's optional. You could default to false

```ts
private _isAPM?: boolean = false
```

Reply:
fixed